packages/atlas/src/hooks/useTimer/useTimer.ts

```diff
@@ -0,0 +1,40 @@
+import { differenceInSeconds, format } from 'date-fns'
+import { useLayoutEffect, useState } from 'react'
+
+import { formatDurationShort } from '@/utils/time'
+
+export const DIFF_THRESHOLD = 3600
+
+export const useTimer = (date?: Date) => {
```

Comment:
LGTM!

Just since `date-fns` accepts `Date | number` arguments and a conversion to `Date` is needed in most case WDYT of accepting a  `Date | number` on this hook too